extensions/skins/conversation_v1/static/conversation.css

```diff
@@ -124,6 +129,46 @@ html, body {
   max-width: 1084px;
 }
 
+.conversation-skin-help-card-container.ng-enter,
+.conversation-skin-help-card-container.ng-leave {
+  opacity: 0;
+  transform: translateX(200px);
+}
+.conversation-skin-help-card-container {
+  opacity: 1;
+  position: fixed;
+  right: 30px;
+  top: 80px;
+  transform: translateX(0);
```

Comment:
This line can be removed, turned out it's always there implicitly and doesn't affect the animation.